src/main.js

```diff
@@ -0,0 +1,102 @@
+// Mobile menu functionality
+import {
+  menuIconMobile, menuLinks, openMenu, closeMenu,
+} from './menu.js';
+
+// Popup window functionality
+import { createModal, projects } from './popup.js';
+
+menuIconMobile.addEventListener('click', openMenu);
+menuLinks.forEach((menuLink) => {
+  menuLink.addEventListener('click', closeMenu);
+});
```

Comment:
- [ ] Great job working on the suggestions from previous reviewers. However, there're some things you still need to work on.
As mentioned by the previous reviewer, the responsiveness of the popup modal is still not in line with the guiding template. The button to close the modal is not showing on both desktop and mobile properly. Kindly take a look and fix the issues. 

### Your Design
[your-design.webm](https://user-images.githubusercontent.com/26597432/211168331-0180d20e-f041-446b-bf2a-431f0c44b4c7.webm)


### Figma Design Desktop
![figma-desktop](https://user-images.githubusercontent.com/26597432/211168432-12acbd5e-64e1-41fe-915e-bd50646e7744.png)

### Figma Design Mobile
![figma-mobile](https://user-images.githubusercontent.com/26597432/211168434-0592a4e1-ff25-41c4-90cb-26352517b7fa.png)